docs/EnzymeIntlHelpers.md

```diff
@@ -0,0 +1,27 @@
+# Enzyme Intl Helpers
+
+Enzyme helpers for fortifying tests that depend on react-intl by decoupling the need for actual translations. See: [react-intl documentation](https://github.com/yahoo/react-intl/wiki/Testing-with-React-Intl#helper-function-1)
+
+## Usage
+
+```jsx
+import { shallowWithIntl, mountWithIntl } from 'terra-toolkit/lib/enzyme-intl';
```

Comment:
Could this just be named export in a new `src/index.js` so that this line just becomes `import {...} from 'terra-toolkit';`

Reply:
I definitely thought about it. I like it, for sure.